src/main/java/com/mraof/minestuck/block/MSBlockShapes.java

```diff
@@ -100,6 +100,7 @@ public class MSBlockShapes
 	public static CustomVoxelShape SMALL_ALCHEMITER = new CustomVoxelShape(new double[]{0,0,0,16,4,16}, new double[]{1,4,1,15,6,15}, new double[]{2,6,2,14,8,14}, new double[]{0.5,4,6.75,1,5,9.25}, new double[]{6.75,4,0.5,9.25,5,1}, new double[]{15,4,6.75,15.5,5,9.25}, new double[]{6.75,4,15,9.25,5,15.5}, new double[]{13.5,4,13.5,15.5,9,15.5}, new double[]{11.5,4,14.5,12.5,15,15.5}, new double[]{13,11,14.5,14,15,15.5}, new double[]{14.5,11,14.5,15.5,15,15.5}, new double[]{12,15,14.5,13.5,15.5,15.5}, new double[]{13.5,10.5,14.5,15,11,15.5}, new double[]{14,15,14,16,17,16});
 	
 	//Misc Machines
+	public static CustomVoxelShape INTELLIBEAM_LAZERSTATION = new CustomVoxelShape(new double[]{1,6,0,6,12,4}, new double[]{1,0,1,6,7,6}, new double[]{4,0,12,11,7,16}, new double[]{0,0,6,8,8,12}, new double[]{8,0,4,15,8,12}, new double[]{-1,7,7,16,10,11}, new double[]{6,7,12,11,16,16}, new double[]{6,15,11,11,21,15}, new double[]{4,20,8,14,26,14}, new double[]{-1,20,7,4,27,15}, new double[]{5,19,5,12,22,10}, new double[]{6,22,6,11,24,8});
```

Comment:
> Looks like it's a bit larger than a block. Are you confident that it is within the size that vanilla can handle correctly for one block?

From the previous PR, still relevant. I'm sure that there's some limit to how large a shape that the game handles without a problem, but I don't know how large that is or whether this is within that. Bringing this up mostly because it's larger than the shape of a fence.

Reply:
Found a potential resource we could use to discuss this in Discord